code/modules/multiz/_HELPERS.dm

```diff
@@ -0,0 +1,243 @@
+/* Multi-Z code was ported from Polaris, which operates under the AGPL v3 license.
+Permission for its use was obtained on 11/12/2017 from Neerti in the Polaris Discord. */
+
+#define isopenspace(A) istype(A, /turf/simulated/open)
+#define isvisiblespace(A) is_type_in_list(A, list(/turf/simulated/open, /turf/simulated/floor/glass))
+#define OPENSPACE_PLANE_START -23
+#define OPENSPACE_PLANE_END -8
+#define OPENSPACE_PLANE -25
+#define OVER_OPENSPACE_PLANE -7
+//#define QDELETED(X) (!X || X.gc_destroyed)  Polaris has this
+
+/obj/effect/landmark/map_data
```

Comment:
In retrospect it seems insane that we are putting this info on a placeable landmark instead of on the code end, set a var on the z level datum used for the map.

Reply:
Moved this to the code end